scripts/2.mutation-classifier.py

```diff
@@ -326,19 +326,45 @@ def get_threshold_metrics(y_true, y_pred):
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
+# ### Top coefficients for covariates model
+
 # In[21]:
 
-# Top standardized coefficients
-(coef_df
-    .query("feature_set == 'full'")
-    .head(10)
-)
+coef_df.query("feature_set == 'covariates'").head(10)
 
 
-# ## Investigate the predictions
+# ### Top coefficients for full model
 
 # In[22]:
 
+coef_df.query("feature_set == 'full'").head(10)
+
+
+# ### Top coefficients for individual _genes_ for expressions model
```

Comment:
Let me know if my comment on the number of PCA components used by each model changes your preferred approach for this. I'm thinking creating one combined dataframe with coefficients and absolute values for both `full` and `expression` and also including two columns for "rank" for full and expression and then showing the top 10 genes for each model as two separate outputs may be the best bet. That may take me a while.

I'll look into it a little more tonight or tomorrow but let me know your thoughts.